flink-python/pyflink/fn_execution/coder_impl.py

```diff
@@ -25,56 +25,331 @@
+                              (True, True, True, False, False, False, False, False),
+                              (True, True, True, False, False, False, False, True),
+                              (True, True, True, False, False, False, True, False),
+                              (True, True, True, False, False, False, True, True),
+                              (True, True, True, False, False, True, False, False),
+                              (True, True, True, False, False, True, False, True),
+                              (True, True, True, False, False, True, True, False),
+                              (True, True, True, False, False, True, True, True),
+                              (True, True, True, False, True, False, False, False),
+                              (True, True, True, False, True, False, False, True),
+                              (True, True, True, False, True, False, True, False),
+                              (True, True, True, False, True, False, True, True),
+                              (True, True, True, False, True, True, False, False),
+                              (True, True, True, False, True, True, False, True),
+                              (True, True, True, False, True, True, True, False),
+                              (True, True, True, False, True, True, True, True),
+                              (True, True, True, True, False, False, False, False),
+                              (True, True, True, True, False, False, False, True),
+                              (True, True, True, True, False, False, True, False),
+                              (True, True, True, True, False, False, True, True),
+                              (True, True, True, True, False, True, False, False),
+                              (True, True, True, True, False, True, False, True),
+                              (True, True, True, True, False, True, True, False),
+                              (True, True, True, True, False, True, True, True),
+                              (True, True, True, True, True, False, False, False),
+                              (True, True, True, True, True, False, False, True),
+                              (True, True, True, True, True, False, True, False),
+                              (True, True, True, True, True, False, True, True),
+                              (True, True, True, True, True, True, False, False),
+                              (True, True, True, True, True, True, False, True),
+                              (True, True, True, True, True, True, True, False),
+                              (True, True, True, True, True, True, True, True))
+
+    null_byte_search_table = (0x80, 0x40, 0x20, 0x10, 0x08, 0x04, 0x02, 0x01)
 
     def __init__(self, field_coders):
         self._field_coders = field_coders
         self._filed_count = len(field_coders)
+        self._complete_byte_num = self._filed_count // 8
```

Comment:
What about renamed to _leading_bytes_num

Reply:
_complete_byte_num -> _leading_complete_bytes_num
_leading_bytes_num -> _remaining_bits_num